shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/data/remote/body/LogInRequestBody.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.avishek.phong.kmmic.data.remote.body
+
+import co.nimblehq.avishek.phong.kmmic.BuildKonfig
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+private const val GRANT_TYPE_PASSWORD = "password"
+
+@Serializable
+data class LogInRequestBody(
+    @SerialName("grant_type")
+    val grantType: String = GRANT_TYPE_PASSWORD,
+    @SerialName("email")
+    val email: String,
+    @SerialName("password")
+    val password: String,
+    @SerialName("client_id")
+    val clientId: String = BuildKonfig.CLIENT_ID,
+    @SerialName("client_secret")
+    val clientSecret: String = BuildKonfig.CLIENT_SECRET,
```

Comment:
Shouldn't we declare these as `nullable`?

Reply:
@phongvhd93 I think we usually declare this as `nullable` so that the app won't crash when API made changes.

Reply:
But it is not the object for API response 🤔 , it is the request body sent from the device.